src/_includes/header-help.html

```diff
@@ -0,0 +1,12 @@
+<section class="main-container main-container--align-center hero layout-grid">
+  <div class="flex-container">
+    <h1>{{ header-title }}</h1>
+    <p>{{ header-paragraph }}</p>
+    <div class="flex">
+      <a href="" class="button">Download start guide</a>
+    </div>
+  </div>
+  <div class="flex-container flex-container--center-content">
+    <img src="{{ image }}" alt="" />
```

Comment:
Should this have an `alt`? The data could be sent along the same as the image src.